lib/semantic-toast.jsx

```diff
@@ -16,6 +16,7 @@ function SemanticToast(props) {
 
     const onDismiss = e => {
         e.stopPropagation();
+        props.onDismiss();
```

Comment:
Can you please destructure `onDismiss` as well?

Reply:
Done!